projects/cn0579/de10nano/system_project.tcl

```diff
@@ -7,6 +7,12 @@ adi_project cn0579_de10nano
 
 source $ad_hdl_dir/projects/common/de10nano/de10nano_system_assign.tcl
 
+## down-rade Critical Warning reated to a asynchronous RAM in DMAC
```

Comment:
Spelling mistake at "downgrade" and "related"

Reply:
Solved in V2.